docs/src/inverse_problems/petab_ode_param_fitting.md

```diff
@@ -300,12 +295,12 @@ measurements = vcat(m1,m2)
 which then can be used as input to `PEtabModel`.
 
 ### Varying parameters between different simulation conditions
-Sometimes, the parameters that are used vary between the different conditions. Consider our catalysis example, if we had performed the experiment twice, using two different enzymes with different catalytic properties, this could have generated such conditions. The two enzymes could e.g. yield different rates ($kP_1$ and $kP_2$) for the `SE --> P + E` reaction, but otherwise be identical. Here, the parameters $kP_1$ and $kP_2$ are unique to their respective conditions. PEtab.jl provides support for cases such as this, and [its documentation](https://sebapersson.github.io/PEtab.jl/stable/Julia_condition_specific/) provided instructions of how to handle them.
+Sometimes, the parameters that are used vary between the different conditions. Consider our catalysis example, if we had performed the experiment twice, using two different enzymes with different catalytic properties, this could have generated such conditions. The two enzymes could e.g. yield different rates ($kP_1$ and $kP_2$) for the `SE --> P + E` reaction, but otherwise be identical. Here, the parameters $kP_1$ and $kP_2$ are unique to their respective conditions. PEtab.jl provides support for cases such as this, and [its documentation](https://sebapersson.github.io/PEtab.jl/stable/petab_cond_specific/) provided instructions of how to handle them.
 
 ## [Additional features: Initial conditions](@id petab_simulation_initial_conditions)
 
 ### [Fitting initial conditions](@id petab_simulation_initial_conditions_fitted)
-Sometimes, initial conditions are uncertain quantities which we wish to fit to the data. This is possible by [defining an initial condition as a parameter](@ref dsl_description_parametric_initial_conditions): 
```

Comment:
any reason to remove this? Seems like a relevant reference?

Reply:
It failed the link check when building before (thought it had been removed), but that might actually been due to something else (as I can actually find the `@id` block in the code). I agree, I will add it back.

Reply:
Seems like an excellent reason to remove it. Yes, the link should be `dsl_advanced_options_parametric_initial_conditions`. Probably got changed while PEtab doc was disabled so this wasn't caught somehow.

Reply:
Ok, seems you found the new link as well